app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :likable, counter_cache: true, polymorphic: true
```

Comment:
Guys, what did we need the `counter_cache: true` for? :)

Reply:
@rvvergara, we plan to use the counter_cache to keep track of likes count automatically instead of having to count the number of records each time. We believe its better for performance.

Reply:
Great! Thanks for the explanation :)